xarray/tests/conftest.py

```diff
@@ -14,9 +14,11 @@ def backend(request):
     return request.param
 
 
-@pytest.fixture(params=["numbagg", "bottleneck"])
+@pytest.fixture(params=["numbagg", "bottleneck", "None"])
```

Comment:
Lets use `None` instead of `"None"` everywhere.

Reply:
Makes total sense. For some reason I had in my mind that these params should be strings. 

Done https://github.com/pydata/xarray/pull/8684/commits/ebfe30fbc3359ed55e6b0bc26bdee5197c54d1ef